docs/docsite/rst/playbooks_prompts.rst

```diff
@@ -27,6 +27,11 @@ Here is a most basic example::
         - name: "favcolor"
           prompt: "what is your favorite color?"
 
+.. note::
+   Prompts for individual "vars_prompt" variables will be skipped for any variable that is already defined on the command line, or when running from a non-interactive session (such as cron or Ansible Tower). See :ref:`_passing_variables_on_the_command_line` in the /Variables/ chapter.
```

Comment:
not defined in command line, but specifically 'extra-vars' which have precedence, so it makes no sense to prompt when those will override anyways. Just to make sure people don't confuse with env vars nor command line switches